src/main/webapp/jsp/instructorFeedbackResultsTop.jsp

```diff
@@ -244,6 +244,11 @@
     <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>" 
         value="<%=data.account.googleId%>">
 </form>
+    <%if(data.selectedSection.equals("All") && showAll==false){%>
+        <div id="statusMessage" class="alert alert-warning" style="margin-top:20px;" >
+            This session seem to have a large number of responses. It is recommended to view the results one section at at time.
```

Comment:
Also, there is already a standardized status message available.
See the line below: <jsp:include page="<%=Const.ViewURIs.STATUS_MESSAGE%>" />
Try to find a way to use the existing system instead of creating the div and duplicating code.